ios/RNSentry.m

```diff
@@ -85,16 +85,19 @@ + (BOOL)requiresMainQueueSetup {
         }
     }
 
+#if TARGET_OS_IPHONE
     // Enable the App start and Frames tracking measurements
     if ([mutableOptions valueForKey:@"enableAutoPerformanceTracking"] != nil) {
         BOOL enableAutoPerformanceTracking = (BOOL)[mutableOptions valueForKey:@"enableAutoPerformanceTracking"];
 
         PrivateSentrySDKOnly.appStartMeasurementHybridSDKMode = enableAutoPerformanceTracking;
         PrivateSentrySDKOnly.framesTrackingMeasurementHybridSDKMode = enableAutoPerformanceTracking;
     }
+#endif
```

Comment:
I think only `framesTrackingMeasurementHybridSDKMode` needs to be protected here, so I'd rather do what the Flutter SDK does, https://github.com/getsentry/sentry-dart/blob/main/flutter/ios/Classes/SentryFlutterPluginApple.swift#L146-L148=